gems/laa_multi_step_forms/docs/Setup.md

```diff
@@ -1 +1,51 @@
-Include details on CSS files and how to get that to work
\ No newline at end of file
+# LAA Multi Step Form setup instriuctions
+
+These instructions are designed to allow you to start a new project
+using using the multi step form engine as a base
+
+## 1. Configurations Steps
+
+### 1. Install the gem(s)
+
+As gemspec files are not able to point to a github repo bothy the below GEM need
+to be added:
+
+```
+gem 'laa_multi_step_forms', path: 'gems/laa_multi_step_forms'
+gem 'hmcts_common_platform', github: 'ministryofjustice/hmcts_common_platform', tag: 'v0.2.0'
+```
+
+NOTE: hmcts_common_platform provides a list fo Court's and is generica enough it has
```

Comment:
generica --> generic enough to be added to the shared gem.

Reply:
I guess you could add this as a dependency in the gemspec? Rather then ask the use to add it but either is good really